docker-compose.yml

```diff
@@ -15,7 +15,8 @@ services:
     ports:
       - "8081:8080"
     volumes:
-      - ./dmas_config.ini:/config.ini
+      - ./dmas_config.ini:/app/config.ini
+      - ./DMAS/models/human_detection/weights/human_classification_weights.pkl:/app/models/human_detection/weights/human_classification_weights.pkl
```

Comment:
This will be grabbed in the build process for DMAS; it's probably best to rebuild the container when it changes rather than bind mount it since it's not really data that changes as it runs

Reply:
(So long as you run the commands before the build process)

Reply:
Ah I see, I didn't realise that docker used a different working directory, fixing now ty